### DISCUSSION TITLE
Smart contract chainlink repo

### DISCUSSION THREAD

**Discussion by Ada-sama** [Cyfrin/foundry-full-course-cu#2477] [General]
HI, please how can i find the smartcontract chainlink repo on github. I am only able to access chainlink feeds here

**Comment by abraj** [comment]
### GitHub repo:
https://github.com/smartcontractkit/chainlink-brownie-contracts

### Install command:
```shell
forge install smartcontractkit/chainlink-brownie-contracts@0.6.1 --no-commit
```

**Reply by Ada-sama** [reply]
Thank you

**Comment by EngrPips** [comment]
Awesome, @Ada-sama, did you get what you are looking for already?

**Reply by Ada-sama** [reply]
yes i have, thankss.

**Reply by EngrPips** [reply]
Awesome.

**Reply by Ada-sama** [reply]
yes. I do have a little issue , if you could help me please. I am trying to push my work to  my repo . But the code I run is giving me an error that the authentication failed, and I have done everything ChatGPT  said I should try out.
 Please if you have any ideas that could help me out I will really appreciate

**Reply by EngrPips** [reply]
Did you watch the lesson where Patrick taught how to push our code to GitHub?

**Reply by Ada-sama** [reply]
Yes that the part I am on now

**Reply by EngrPips** [reply]
I will say watch that part carefully and follow all his instructions that was what I watched, and I didn't have an issue pushing my code to GitHub.

**Reply by Ada-sama** [reply]
Alright thank  you

**Reply by EngrPips** [reply]
Anytime
